controller/PciManager.php

```diff
@@ -227,20 +224,17 @@ public function add()
 
     /**
      * Export PCI zip package with all runtime, creator & manifest files
-     * Not used yet
-     * @todo Download path e.q. $path
      */
     public function export()
     {
         //as upload may be called multiple times, we remove the session lock as soon as possible
         session_write_close();
-
+        $requestPayload = $this->getPsrRequest()->getQueryParams();
         try {
-            if (!$this->hasRequestParameter('typeIdentifier')) {
-                throw new PortableElementException('Type identifier parameter missing.');
+            if (!isset($requestPayload['typeIdentifier'], $requestPayload['pciIdentifier'])) {
+                throw new PortableElementException('PCI parameter missing.');
             }
-            $identifier = $this->getRequestParameter('typeIdentifier');
-            $path = $this->getService()->export(PciModel::PCI_IDENTIFIER, $identifier);
+            $path = $this->getService()->export($requestPayload['pciIdentifier'], $requestPayload['typeIdentifier']);
```

Comment:
Should `$requestPayload['pciIdentifier']` be mandatory as well?

Reply:
Now it's should be mandatory.